syntax="proto3";

package ocean_mprpc;

message ResultCode{
    int32 errcode =1;
    bytes errmsg =2;
}